src/foam/graphics/TreeGraph.js

```diff
@@ -48,9 +48,8 @@
 
        // List for 'click' events to expand/collapse Nodes.
        this.canvas.on('click', function(e) {
-
          var x = e.layerX+this.nodeWidth/2, y = e.layerY;
-         var c = this.root.findFirstChildAt(x, y);
+         var c = this.root.findFirstChildAt(x/this.canvas.cview.scaleX, y/this.canvas.cview.scaleY);
```

Comment:
Couldn't/shouldn't the findFirstChildAt() method handle the scaling?

Reply:
Doesn't appear as the X and Y values get directly manipulated upon changing the scale of the canvas and so when changing the scale, the X and Y values will be whatever it is at scale = 1, X won't become X * scaleX. And after inspecting the findFirstchildAt function there is no adjustment made for the scale at all as it uses the hard X and Y values:
<img width="1680" alt="Screen Shot 2019-08-15 at 8 27 59 PM" src="https://user-images.githubusercontent.com/26717171/63135326-71b47a00-bf9b-11e9-9474-a59a7381d2d1.png">

Reply:
I didn't mean that findFirstChildAt() does take the scale into account. I meant that it should. It would make a better interface if it just handled it for you.

Reply:
Done.